src/measure_theory/constructions/borel_space.lean

```diff
@@ -1307,6 +1338,18 @@ begin
     exact measurable_set.bInter hs (λ i hi, measurable_set_le (hf i) measurable_const) }
 end
 
+lemma measurable_csupr_subtype {α' : Type*}
```

Comment:
Do you still need this lemma, and all the other csupr lemmas below like supr_subtype_eq_sup'? I had the impression that they were not used any more.

Reply:
I don't need those lemmas anymore but I thought they would be nice to have anyway. Should I remove them?

Reply:
In particular, since `⨆ x ∈ s, f x` is a bad spelling, if we want to the a csupr of a not necessarily finite set, we will use the subtype spelling.

Reply:
I would just drop them from the PR for now, and add them later if/when they are needed.